tests/spread/general/dependency-check-missing-lib/snaps/missing-lib/snap/snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: missing-lib
+version: 'test1'
+summary: test
+description: test
+grade: stable
+confinement: strict
+
+parts:
+  pciutils:
+    plugin: nil
+    build-packages:
+      - pciutils
+    override-build: |
+      mkdir -p $SNAPCRAFT_PART_INSTALL/usr/bin
+      install -m 0755 /usr/bin/lspci $SNAPCRAFT_PART_INSTALL/usr/bin
```

Comment:
install -D and you can remove the line above

Reply:
did not know of that, done!